lib/Plack/Middleware/OpenTelemetry.pm

```diff
@@ -2,10 +2,12 @@ package Plack::Middleware::OpenTelemetry;
 
 # ABSTRACT: Plack middleware to setup OpenTelemetry tracing
 
-use v5.36.0;
 use strict;
 use warnings;
+
 use feature 'signatures';
+no warnings 'experimental::signatures';
```

Comment:
```suggestion
use experimental 'signatures';
```

This distribution depends on [OpenTelemetry](https://metacpan.org/pod/OpenTelemetry), which requires Perl 5.30 or higher. `signatures` was added to the experimental bundle in 5.20 and they've been largely stable in their current form since 5.28.

Reply:
Thank you both; I uploaded a new version with this change.